packages/init/src/steps/install-required-dependencies.ts

```diff
@@ -10,7 +31,7 @@ export async function installRequiredDependencies(
   const installDevDependencyCommand = yarnLock
     ? "yarn add -D"
     : "npm install --save-dev";
-  const installDependencyCommand = yarnLock ? "yarn add" : "npm install";
+  const installDependencyCommand = yarnLock ? "yarn add" : "npm install --save";
 
   const spinner = Utils.spinner("Downloading Prismic Visual Builder");
```

Comment:
OPT: I think it could be good to split this function into two
- installDevDeps
- installDeps
- installRequiredDeps (that call the two above)

This would declare two functions clear that each display their owner spinner or create their own command based on the packageManager passed as parameter.
I think it would make the code easier to test.
But I acknowledge that this would require work on the test and all.
That's why my comment is optional 😄